apps/internal/base/internal/storage/items_test.go

```diff
@@ -204,13 +203,7 @@ func TestAppMetaDataMarshal(t *testing.T) {
 }
 
 func TestContractUnmarshalJSON(t *testing.T) {
-	jsonFile, err := os.Open(testFile)
-	if err != nil {
-		panic(err)
-	}
-	defer jsonFile.Close()
```

Comment:
Was this inadvertently removed?

Reply:
No, I changed `os.Open -> ioutil.ReadAll` to simply `os.ReadFile`.

Reply:
Oh yes, I read that wrong.